api/app/GraphQL/Validators/Mutation/ChangeApplicationSuspendedAtValidator.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\GraphQL\Validators\Mutation;
+use Database\Helpers\ApiEnums;
+use Illuminate\Validation\Rule;
+use Nuwave\Lighthouse\Validation\Validator;
+
+final class ChangeApplicationSuspendedAtValidator extends Validator
+{
+    /**
+     * Return the validation rules.
+     *
+     * @return array<string, array<mixed>>
+     */
+    public function rules(): array
+    {
+        return [
+            'submitted_at' => ['required'],
```

Comment:
Right now we don't have a way to submit at an arbitrary time but it would be safer if this was a "submitted_at < now" comparison instead of just "not null".

Reply:
Changed with [994d23e](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5832/commits/994d23e17c80651d3452b511733aed38bfb2ebbb).